src/components/paymentTable/paymentTable.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import './paymentTable.scss';
+import '../../assets/main.css';
+import { useTranslation } from 'react-i18next';
+import booksRender from './booksRender';
+
+import {
+  MDBContainer,
+  MDBRow,
+  MDBCol,
+  MDBTable,
+  MDBTableBody,
+  MDBTableHead,
+} from 'mdbreact';
+
+export default function PaymentTable() {
+  const { t } = useTranslation();
+  return (
+    <MDBContainer>
+      <MDBRow>
+        <MDBCol md="12">
+          <div className="text-center">
+            <p className="h5 mt-5 text-gray-800">
+              {t('PaymentPage.booksOrder')}
+            </p>
+            <hr className="border-4 my-3 mx-4 bg-primary"></hr>
+          </div>
+          <MDBTable>
+            <MDBTableHead>
+              <tr>
+                <th></th>
+                <th>{t('PaymentPage.books')}</th>
+                <th>{t('PaymentPage.copies')}</th>
+                <th>{t('PaymentPage.price')}</th>
+              </tr>
+            </MDBTableHead>
+            <MDBTableBody>
+              {booksRender()}
```

Comment:
Why don't you make it as a component so it can manage it state for the future because we going to have a state of selected books?   
I f you going to make a component don't forget to start the name with a capital letter

Reply:
it's already a component you just need to change the file name and make the first letter upper case as well as the function and the import statement and then you can call it as a component `BooksRender`